### PR TITLE
Update digitalmarketplace-scripts jobs to use invoke instead of make

### PR DESCRIPTION
**Pull request by lfdebrux**
alphagov/digitalmarketplace-scripts#649 replaced make with invoke in the scripts repo, we need to update our job definitions to reflect this.

### CODE REVIEW

File: job_definitions/generate_upload_and_notify_counterpart_signature_pages.yml [hunk 1]
```diff
@@ -64,18 +64,17 @@
...
+                  invoke requirements
                 ''')
               }
             }
             stage('Generate') {
               dir('digitalmarketplace-scripts') {
                 sh('''
                   [ "$DRY_RUN" = "true" ] && FLAGS="$FLAGS --dry-run"
-                  
+
```
Comment (gidsg) [inline job_definitions/generate_upload_and_notify_counterpart_signature_pages.yml:77]: very tiny but is this whitespace change intended?

**Review by gidsg — APPROVED** [review]
Looks fine to me, have you tested on Jenkins?

File: job_definitions/generate_upload_and_notify_counterpart_signature_pages.yml (hunk 1 above)
Comment (lfdebrux) [reply job_definitions/generate_upload_and_notify_counterpart_signature_pages.yml:77]: Yes, my editor was complaining about it.

**Comment by lfdebrux** [comment]
@gidsg tested it just now, it turns out the pip bin directory isn't in the Jenkins user's path, so I need to make a few more changes.

**Comment by lfdebrux** [comment]
I've pushed some changes to make ~jenkins/.local/bin available in PATH, so this works now:

https://ci.marketplace.team/job/generate-upload-and-notify-counterpart-signature-pages/1461/console

**Pull request merged**